mcs/class/System.Data/System.Data-net_4_x.csproj

```diff
@@ -49,6 +49,26 @@
   </PropertyGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <ItemGroup>
+    <Compile Include="..\..\..\external\corefx\src\System.Data.Common\src\System\Data\SQLTypes\INullable.cs" />

```

Comment:
As mentioned on the top of the file, the .csproj's are autogenerated so you don't really need to modify it manually. We have a bot that updates them on demand (the csproj's are not used by our build).